packages/Algernon.package/ALGCore.class/instance/mouseDown..st

```diff
@@ -0,0 +1,5 @@
+initialization
+mouseDown: evt 
```

Comment:
Is there a reason the ResultItems or the ResultList can't handle mousedown themselves?

Reply:
Afaik we need the ALGCore to have Mouse-Focus to support the moving of Algernon

Reply:
Maybe we can support moving algernon only by clicking on the breadcrumbs and not the results?